labman/db/tests/test_process.py

```diff
@@ -1694,6 +1771,2312 @@ def test_generate_prep_information(self):
+                                                    input_row_index=7,
+                                                    input_col_index=8),
+    GDNAPlateCompressionProcess.InterleavedPosition(output_row_index=0,
+                                                    output_col_index=19,
+                                                    input_plate_order_index=1,
+                                                    input_row_index=0,
+                                                    input_col_index=9),
+    GDNAPlateCompressionProcess.InterleavedPosition(output_row_index=2,
+                                                    output_col_index=19,
+                                                    input_plate_order_index=1,
+                                                    input_row_index=1,
+                                                    input_col_index=9),
+    GDNAPlateCompressionProcess.InterleavedPosition(output_row_index=4,
+                                                    output_col_index=19,
+                                                    input_plate_order_index=1,
+                                                    input_row_index=2,
+                                                    input_col_index=9),
+    GDNAPlateCompressionProcess.InterleavedPosition(output_row_index=6,
+                                                    output_col_index=19,
+                                                    input_plate_order_index=1,
+                                                    input_row_index=3,
+                                                    input_col_index=9),
+    GDNAPlateCompressionProcess.InterleavedPosition(output_row_index=8,
+                                                    output_col_index=19,
+                                                    input_plate_order_index=1,
+                                                    input_row_index=4,
+                                                    input_col_index=9),
+    GDNAPlateCompressionProcess.InterleavedPosition(output_row_index=10,
+                                                    output_col_index=19,
+                                                    input_plate_order_index=1,
+                                                    input_row_index=5,
+                                                    input_col_index=9),
+    GDNAPlateCompressionProcess.InterleavedPosition(output_row_index=12,
+                                                    output_col_index=19,
+                                                    input_plate_order_index=1,
+                                                    input_row_index=6,
+                                                    input_col_index=9),
+    GDNAPlateCompressionProcess.InterleavedPosition(output_row_index=14,
+                                                    output_col_index=19,
+                                                    input_plate_order_index=1,
+        
```

Comment:
I'm assuming this is hard coded from the wetlab?

Reply:
Yes; I've added an explanatory comment.